x-pack/test/detection_engine_api_integration/utils.ts

```diff
@@ -538,6 +578,25 @@ export const createSignalsIndex = async (
   );
 };
 
+/**
+ * Creates the preview signals index for use inside of beforeEach blocks of tests
+ * This will retry 20 times before giving up and hopefully still not interfere with other tests
+ * @param supertest The supertest client library
+ */
+export const createPreviewSignalsIndex = async (
+  supertest: SuperTest.SuperTest<SuperTest.Test>,
+  log: ToolingLog
+): Promise<void> => {
+  await countDownTest(
+    async () => {
+      await supertest.post(DETECTION_ENGINE_RULES_PREVIEW_INDEX_URL).set('kbn-xsrf', 'true').send();
```

Comment:
will there be a privilege problem here?

Reply:
shouldn't be, but we also might not need this at all actually since we switched to the rule registry executors and it _should_ create this index on its own

Reply:
Please verify this and delete if possible 👍